File: services/src/main/java/com/liferay/mobile/sdk/v62/shoppingorder/ShoppingOrderService.java
package com.liferay.mobile.sdk.v62.shoppingorder;

import com.liferay.mobile.sdk.Call;
import com.liferay.mobile.sdk.annotation.Param;
import com.liferay.mobile.sdk.annotation.Path;
import com.liferay.mobile.sdk.http.Response;
import java.lang.String;
import org.json.JSONObject;

@Path("/shoppingorder")
public interface ShoppingOrderService {
  @Path("/complete-order")
  Call<Response> completeOrder(@Param(name = "groupId") long groupId, @Param(name = "number") String number, @Param(name = "ppTxnId") String ppTxnId, @Param(name = "ppPaymentStatus") String ppPaymentStatus, @Param(name = "ppPaymentGross") double ppPaymentGross, @Param(name = "ppReceiverEmail") String ppReceiverEmail, @Param(name = "ppPayerEmail") String ppPayerEmail, @Param(name = "serviceContext", className = "com.liferay.portal.service.ServiceContext") JSONObject serviceContext);

  @Path("/delete-order")
  Call<Response> deleteOrder(@Param(name = "groupId") long groupId, @Param(name = "orderId") long orderId);

  @Path("/get-order")
  Call<JSONObject> getOrder(@Param(name = "groupId") long groupId, @Param(name = "orderId") long orderId);

  @Path("/send-email")
  Call<Response> sendEmail(@Param(name = "groupId") long groupId, @Param(name = "orderId") long orderId, @Param(name = "emailType") String emailType, @Param(name = "serviceContext", className = "com.liferay.portal.service.ServiceContext") JSONObject serviceContext);

  @Path("/update-order")
  Call<JSONObject> updateOrder(@Param(name = "groupId") long groupId, @Param(name = "orderId") long orderId, @Param(name = "billingFirstName") String billingFirstName, @Param(name = "billingLastName") String billingLastName, @Param(name = "billingEmailAddress") String billingEmailAddress, @Param(name = "billingCompany") String billingCompany, @Param(name = "billingStreet") String billingStreet, @Param(name = "billingCity") String billingCity, @Param(name = "billingState") String billingState, @Param(name = "billingZip") String billingZip, @Param(name = "billingCountry") String billingCountry, @Param(name = "billingPhone") String billingPhone, @Param(name = "shipToBilling") boolean shipToBilling, @Param(name = "shippingFirstName") String shippingFirstName, @Param(name = "shippingLastName") String shippingLastName, @Param(name = "shippingEmailAddress") String shippingEmailAddress, @Param(name = "shippingCompany") String shippingCompany, @Param(name = "shippingStreet") String shippingStreet, @Param(name = "shippingCity") String shippingCity, @Param(name = "shippingState") String shippingState, @Param(name = "shippingZip") String shippingZip, @Param(name = "shippingCountry") String shippingCountry, @Param(name = "shippingPhone") String shippingPhone, @Param(name = "ccName") String ccName, @Param(name = "ccType") String ccType, @Param(name = "ccNumber") String ccNumber, @Param(name = "ccExpMonth") int ccExpMonth, @Param(name = "ccExpYear") int ccExpYear, @Param(name = "ccVerNumber") String ccVerNumber, @Param(name = "comments") String comments);

  @Path("/update-order")
  Call<JSONObject> updateOrder(@Param(name = "groupId") long groupId, @Param(name = "orderId") long orderId, @Param(name = "ppTxnId") String ppTxnId, @Param(name = "ppPaymentStatus") String ppPaymentStatus, @Param(name = "ppPaymentGross") double ppPaymentGross, @Param(name = "ppReceiverEmail") String ppReceiverEmail, @Param(name = "ppPayerEmail") String ppPayerEmail);
}

File: windows/Liferay.SDK/Service/V62/ShoppingOrder/ShoppingOrderService.cs
/**
 * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
 *
 * This library is free software; you can redistribute it and/or modify it under
 * the terms of the GNU Lesser General Public License as published by the Free
 * Software Foundation; either version 2.1 of the License, or (at your option)
 * any later version.
 *
 * This library is distributed in the hope that it will be useful, but WITHOUT
 * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
 * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
 * details.
 */

using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Threading.Tasks;

namespace Liferay.SDK.Service.V62.ShoppingOrder
{
	public class ShoppingOrderService : ServiceBase
	{
		public ShoppingOrderService(ISession session)
			: base(session)
		{
		}

		public async Task CompleteOrderAsync(long groupId, string number, string ppTxnId, string ppPaymentStatus, double ppPaymentGross, string ppReceiverEmail, string ppPayerEmail, JsonObjectWrapper serviceContext)
		{
			var _parameters = new JsonObject();

			_parameters.Add("groupId", groupId);
			_parameters.Add("number", number);
			_parameters.Add("ppTxnId", ppTxnId);
			_parameters.Add("ppPaymentStatus", ppPaymentStatus);
			_parameters.Add("ppPaymentGross", ppPaymentGross);
			_parameters.Add("ppReceiverEmail", ppReceiverEmail);
			_parameters.Add("ppPayerEmail", ppPayerEmail);
			this.MangleWrapper(_parameters, "serviceContext", "com.liferay.portal.service.ServiceContext", serviceContext);

			var _command = new JsonObject()
			{
				{ "/shoppingorder/complete-order", _parameters }
			};

			await this.Session.InvokeAsync(_command);
		}

		public async Task DeleteOrderAsync(long groupId, long orderId)
		{
			var _parameters = new JsonObject();

			_parameters.Add("groupId", groupId);
			_parameters.Add("orderId", orderId);

			var _command = new JsonObject()
			{
				{ "/shoppingorder/delete-order", _parameters }
			};

			await this.Session.InvokeAsync(_command);
		}

		public async Task<dynamic> GetOrderAsync(long groupId, long orderId)
		{
			var _parameters = new JsonObject();

			_parameters.Add("groupId", groupId);
			_parameters.Add("orderId", orderId);

			var _command = new JsonObject()
			{
				{ "/shoppingorder/get-order", _parameters }
			};

			var _obj = await this.Session.InvokeAsync(_command);

			return (dynamic)_obj;
		}

		public async Task SendEmailAsync(long groupId, long orderId, string emailType, JsonObjectWrapper serviceContext)
		{
			var _parameters = new JsonObject();

			_parameters.Add("groupId", groupId);
			_parameters.Add("orderId", orderId);
			_parameters.Add("emailType", emailType);
			this.MangleWrapper(_parameters, "serviceContext", "com.liferay.portal.service.ServiceContext", serviceContext);

			var _command = new JsonObject()
			{
				{ "/shoppingorder/send-email", _parameters }
			};

			await this.Session.InvokeAsync(_command);
		}

		public async Task<dynamic> UpdateOrderAsync(long groupId, long orderId, string billingFirstName, string billingLastName, string billingEmailAddress, string billingCompany, string billingStreet, string billingCity, string billingState, string billingZip, string billingCountry, string billingPhone, bool shipToBilling, string shippingFirstName, string shippingLastName, string shippingEmailAddress, string shippingCompany, string shippingStreet, string shippingCity, string shippingState, string shippingZip, string shippingCountry, string shippingPhone, string ccName, string ccType, string ccNumber, int ccExpMonth, int ccExpYear, string ccVerNumber, string comments)
		{
			var _parameters = new JsonObject();

			_parameters.Add("groupId", groupId);
			_parameters.Add("orderId", orderId);
			_parameters.Add("billingFirstName", billingFirstName);
			_parameters.Add("billingLastName", billingLastName);
			_parameters.Add("billingEmailAddress", billingEmailAddress);
			_parameters.Add("billingCompany", billingCompany);
			_parameters.Add("billingStreet", billingStreet);
			_parameters.Add("billingCity", billingCity);
			_parameters.Add("billingState", billingState);
			_parameters.Add("billingZip", billingZip);
			_parameters.Add("billingCountry", billingCountry);
			_parameters.Add("billingPhone", billingPhone);
			_parameters.Add("shipToBilling", shipToBilling);
			_parameters.Add("shippingFirstName", shippingFirstName);
			_parameters.Add("shippingLastName", shippingLastName);
			_parameters.Add("shippingEmailAddress", shippingEmailAddress);
			_parameters.Add("shippingCompany", shippingCompany);
			_parameters.Add("shippingStreet", shippingStreet);
			_parameters.Add("shippingCity", shippingCity);
			_parameters.Add("shippingState", shippingState);
			_parameters.Add("shippingZip", shippingZip);
			_parameters.Add("shippingCountry", shippingCountry);
			_parameters.Add("shippingPhone", shippingPhone);
			_parameters.Add("ccName", ccName);
			_parameters.Add("ccType", ccType);
			_parameters.Add("ccNumber", ccNumber);
			_parameters.Add("ccExpMonth", ccExpMonth);
			_parameters.Add("ccExpYear", ccExpYear);
			_parameters.Add("ccVerNumber", ccVerNumber);
			_parameters.Add("comments", comments);

			var _command = new JsonObject()
			{
				{ "/shoppingorder/update-order", _parameters }
			};

			var _obj = await this.Session.InvokeAsync(_command);

			return (dynamic)_obj;
		}

		public async Task<dynamic> UpdateOrderAsync(long groupId, long orderId, string ppTxnId, string ppPaymentStatus, double ppPaymentGross, string ppReceiverEmail, string ppPayerEmail)
		{
			var _parameters = new JsonObject();

			_parameters.Add("groupId", groupId);
			_parameters.Add("orderId", orderId);
			_parameters.Add("ppTxnId", ppTxnId);
			_parameters.Add("ppPaymentStatus", ppPaymentStatus);
			_parameters.Add("ppPaymentGross", ppPaymentGross);
			_parameters.Add("ppReceiverEmail", ppReceiverEmail);
			_parameters.Add("ppPayerEmail", ppPayerEmail);

			var _command = new JsonObject()
			{
				{ "/shoppingorder/update-order", _parameters }
			};

			var _obj = await this.Session.InvokeAsync(_command);

			return (dynamic)_obj;
		}
	}
}